src/applications/vaos/services/slot/transformers.js

```diff
@@ -0,0 +1,15 @@
+/**
+ * Transforms /vaos/facilities/${facilityId}/available_appointments?type_of_care_id=${typeOfCareId}&clinic_ids[]=${clinicId}&start_date=${startDate}&end_date=${endDate} to
+ * /Slot?status=free
+ *
+ * @export
+ * @param {Array} slots A list of appointment slots from var-resources
+ * @returns {Array} A list of Slot resources in FHIR format
+ */
+export function transformSlots(slots) {
+  return slots.map(slot => ({
+    status: 'free',
+    start: slot.startDateTime,
```

Comment:
I think we'll likely get the dates back in `2019-01-01T14:00:00.000Z` format, based on Cerner sandbox. We could parse and convert them to that format here.